packages/web-reporter/writeReport.ts

```diff
@@ -1,11 +1,55 @@
 import fs from "fs";
+import { TestCaseResult } from "@perf-profiler/types";
+
+// We should have better management of the time interval over the repo
+const MEASURE_INTERVAL = 500;
+
+const assertTimeIntervalMultiple = (n: number) => {
+  if (n % MEASURE_INTERVAL !== 0) {
+    throw new Error(
```

Comment:
I think at this point it makes more sense to do this, as skipping 600ms for instance would not make the report start at 600ms but at 1000ms

Reply:
It's ok for me :)